DEPS

```diff
@@ -45,7 +45,7 @@ vars = {
   # Dart is: https://github.com/dart-lang/sdk/blob/main/DEPS.
   # You can use //tools/dart/create_updated_flutter_deps.py to produce
   # updated revision list of existing dependencies.
-  'dart_revision': '410431ef75750b886482f85e74d73705b0c416b2',
+  'dart_revision': 'e17e752739cf64fd6db9471fe0062b37f9b0e2de',
```

Comment:
I think you want 204b67e5a029de64899154068cf1099b4f7db1f1 as that's the one with breaking https://dart.googlesource.com/sdk/+/2acf0360232d481549b545f2de0a5a6237bacc42

Reply:
Gotcha - updated to 204b67e5a029de64899154068cf1099b4f7db1f1.